docs/helpers/SeleniumWebdriver.md

```diff
@@ -7,7 +7,7 @@ title: SeleniumWebdriver
 
 ## SeleniumWebdriver
```

Comment:
Should be remove this helper, cause we don't use it anymore.

Reply:
Ok!